arches/app/media/js/views/components/search/map-filter.js

```diff
@@ -293,7 +299,7 @@ define([
                         var geojsonFC = self.filter.feature_collection();
                         var extent = geojsonExtent(geojsonFC);
                         var bounds = new this.mapboxgl.LngLatBounds(extent);
-                        self.map().fitBounds(bounds, {
+                        self.mapFitBounds(bounds, {
                             padding: self.buffer()
                         });
```

Comment:
Why is no force value set here but is set in line 509?

Reply:
Using `force=true` here is so that when an explicit call to `zoomToGeoJSON` is made, it ignores this setting. This function is bound to the results map link.

I've used mapFitBound function here to keep consistency.